blog.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>OUR BLOG</title>
+</head>
+
+<body>
+  <header>
+    <a href="index.html">
+      <img src="..." alt="Binabox Logo" />
+    </a>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="blog.html">Blog</a></li>
+        <li><a href="...">Catalog</a></li>

```

Comment:
Using # in links is a common practice in web development, and here’s why it can be useful when creating a link that doesn’t have a destination yet, developers often use `href="#"` as a temporary placeholder.

`<a href="#">Click Me</a>`

Reply:
Pls pay attention to this comment